tests/integration/backend_test.py

```diff
@@ -376,14 +376,14 @@ def test_classical(
     c.add_c_setreg(23, a)
     c.add_c_copyreg(a, b)
 
-    c.add_classicalexpbox_register(a + b, d)
-    c.add_classicalexpbox_register(a - b, d)
+    c.add_classicalexpbox_register(a + b, d)  # type: ignore
+    c.add_classicalexpbox_register(a - b, d)  # type: ignore
```

Comment:
```suggestion
    c.add_classicalexpbox_register(a + b, d.to_list())
    c.add_classicalexpbox_register(a - b, d.to_list())
```
Not sure how these slipped through the net before. I think I'd rather make them type-correct like this than add `type: ignore` comments, since the latter can mask actual bugs. I agree the old version is easier to read, but we can probably fix up the typing in pytket so that it is also accepted.

Reply:
I was not sure if this might change the way this circuit is now handled. It might be possible that the left side has less or more bits than the register we are writing to?

Reply:
> I was not sure if this might change the way this circuit is now handled. It might be possible that the left side has less or more bits than the register we are writing to?

It won't make any functional difference. `add_classical_expbox` just passes the whole register as a `std::vector<Bit>` to the binders.